rts/Rml/SolLua/bind/Document.cpp

```diff
@@ -111,7 +114,11 @@ namespace Rml::SolLua
 		);
 
 		/***
-		 * @alias RmlFocusFlag "None" | "Document" | "Keep" | "Auto"
+		 * @enum RmlFocusFlag 
```

Comment:
This one still needs use of `@field` for the variants. This will be interpreted as an enum table with no fields and a multiline description of `| "None" \n| "...`.